### PR TITLE
Dev/xygu/20210318/localization and hotswap samples

### PR DESCRIPTION
**Pull request by Xiaoy312**
re: unoplatform/uno#5437

### CODE REVIEW

**Comment by Xiaoy312** [comment]
note for reviewer(s): check the "docs: ..." commits for the actual changes made to the projects.

File: UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/LanguageSettings.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+		private void SetAppLanguage(object sender, RoutedEventArgs e)
+		{
+			if ((sender as Button)?.Tag is string tag)
+			{
+				// Change the app language
+				var culture = new System.Globalization.CultureInfo(tag);
+
+#if WINDOWS_UWP
+				Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride = culture.IetfLanguageTag;
```
Comment (jeromelaban) [inline UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/LanguageSettings.xaml.cs:38]: Isn't this working in 3.6 ? @carldebilly  you added support for this, right ?

Comment (Xiaoy312) [reply UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/LanguageSettings.xaml.cs:38]: @jeromelaban It should be, but this is using 3.5.1 atm.
I wasn't sure about to complicating the guide with package update stuffs. But, I guess 3.6.0 is on the horizon?

Comment (davidjohnoliver) [reply UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/LanguageSettings.xaml.cs:38]: The plan is to push the new docs content at the same time as 3.6 is released ;)

File: UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/Strings/fr/Resources.resw [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    <value>Paramètres de langue</value>
+  </data>
+  <data name="MainPage_GotoNextPage.Content" xml:space="preserve">
+    <value>Page suivante</value>
+  </data>
+  <data name="MainPage_IntroText.Text" xml:space="preserve">
+    <value>Bonjour Uno</value>
+  </data>
+  <data name="Page1_GoBack.Content" xml:space="preserve">
+    <value>Returner</value>
```
Comment (davidjohnoliver) [inline UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/Strings/fr/Resources.resw:139]:
```suggestion
    <value>Retourner</value>
```

File: UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/Strings/fr/Resources.resw [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="LanguageSettings_GoBack.Content" xml:space="preserve">
+    <value>Returner</value>
```
Comment (davidjohnoliver) [inline UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Shared/Strings/fr/Resources.resw:121]:
```suggestion
    <value>Retourner</value>
```

**Review by davidjohnoliver — APPROVED** [review]

**Pull request merged**